Rakefile

```diff
@@ -13,8 +13,10 @@ PUPPET_SPEC = File.join(PROJECT_ROOT, 'ruby', 'puppet', 'spec')
 FACTER_LIB = File.join(PROJECT_ROOT, 'ruby', 'facter', 'lib')
 PUPPET_SERVER_RUBY_SRC = File.join(PROJECT_ROOT, 'src', 'ruby', 'puppetserver-lib')
 PUPPET_SUBMODULE_PATH = File.join('ruby','puppet')
-CURRENT_BRANCH = 'master'
-JENKINS_BRANCH = CURRENT_BRANCH
+# Branch of puppetserver for which to update submodule pins
+CURRENT_BRANCH = ENV['PUPPETSERVER_BRANCH'] || '5.1.x'
+# Branch of puppet-agent to track for passing puppet SHA
+JENKINS_BRANCH = ENV['PUPPET_AGENT_BRANCH'] || '5.3.x'
```

Comment:
I think your environment names are much better names for the constants as well, can we go with those everywhere?